Tests/XCUITests/SearchTest.swift

```diff
@@ -255,4 +269,22 @@ class SearchTests: BaseTestCase {
             XCTAssert(keyboardsCount > 0, "The keyboard is not shown")
         }
     }
+
+    func testOpenTabsInSearchSuggestions() {
+        // Go to "Youtube.com" and check the page displays correctly
+        navigator.openURL("https://www.youtube.com")
```

Comment:
I see this is the website used in the test, but was wondering if we could use one served by our local web server so that it is more stable and takes less to load. See for example:
https://github.com/mozilla-mobile/firefox-ios/blob/820c78512470ab47ef9d661616f5175cef8bd52e/Tests/XCUITests/FindInPageTest.swift#L22

Sometimes using these pages do not work depending on the tests, but worth trying if you agree.